src/generator/generator.ts

```diff
@@ -431,27 +224,17 @@ export class Generator {
   }
 
   private async generate(apiDiscoveryUrl: string, schema: Schema) {
-    this.logResult(apiDiscoveryUrl, 'Discovery doc request complete.');
-    const tasks = new Array<() => Promise<void>>();
-    this.getFragmentsForSchema(
-      apiDiscoveryUrl,
-      schema,
-      `${FRAGMENT_URL}${schema.name}/${schema.version}/0/${schema.name}`,
-      tasks
-    );
-
-    // e.g. apis/drive/v2.js
-    const exportFilename = path.join(
-      srcPath,
-      'apis',
-      schema.name,
-      schema.version + '.ts'
-    );
-    this.logResult(apiDiscoveryUrl, 'Downloading snippets...');
-    await Promise.all(tasks.map(t => t()));
     this.logResult(apiDiscoveryUrl, 'Generating APIs...');
+    const apiPath = path.join(srcPath, 'apis', schema.name);
+    const exportFilename = path.join(apiPath, schema.version + '.ts');
     await mkdirp(path.dirname(exportFilename));
+    // populate the `method.fragment` property with samples
+    addFragments(schema);
+    // generate the API (ex: src/apis/youtube/v3.ts)
     await this.render(API_TEMPLATE, {api: schema}, exportFilename);
+    // generate samples on disk at:
+    // src/apis/<service>/samples/<version>/<method>
+    // generateSamples(apiPath, schema);
```

Comment:
commented out on purpose?

Reply:
Yes!  The code works, but that's what write the samples to disk.  I don't want to do that during this first pass.